src/alfasim_sdk/_internal/alfacase/alfacase_to_case.py

```diff
@@ -1508,6 +1524,7 @@ def load_trends_output_description(
         "global_trends": load_global_trend_description,
         "overall_pipe_trends": load_overall_pipe_trend_description,
         "separator_trends": load_separator_trend_description,
+        "controller_trends": load_controller_trend_description,
```

Comment:
@dettmerramon is adding autoloaders support to dict of instances in https://github.com/ESSS/alfasim-sdk/pull/210.
Maybe you 2 could synchronize the efforts and use autoloaders for this.

Reply:
He already merged his work into the rb branch for `1.10`, maybe you can try if it's released and merged into master later =)

See how it can reduce code (my personal wish is not even have those shallow explicit loaders):
https://github.com/ESSS/alfasim-sdk/pull/210/files#diff-baad9c5e2626aa8061d6d114d418a86b6648f7249e03a80c289b9e2a6ec3e304R1750-R1753